wpanusb.c

```diff
@@ -56,6 +57,17 @@ static int wpanusb_control_send(struct wpanusb *wpanusb, unsigned int pipe,
 			       0, 0, data, size, 1000);
 }
 
+static int wpanusb_control_recv(struct wpanusb *wpanusb, u8 request, void *data, u16 size)
+{
+	struct usb_device *udev = wpanusb->udev;
+
+	usb_control_msg(udev, usb_sndctrlpipe(udev, 0), request, VENDOR_OUT,
+			       0, 0, data, size, 1000);
```

Comment:
data should be NULL and size should be 0 here because no data is being sent with the request

Reply:
fixed in updated commit